CS3450.TooRisky/model/Country.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CS3450.TooRisky.model
+{
+    /// <summary>
+    /// Represents a country in the game.
+    /// </summary>
+    public class Country
+    {
+        /// <summary>
+        /// The name of this country.
+        /// </summary>
+        public string Name = "";
+
+        /// <summary>
+        /// A list of the names of the adjacent countries.
+        /// </summary>
+        public List<string> AdjacentCountryNames = new List<string>();
+
+        /// <summary>
+        /// The number of the units on this country.
+        /// </summary>
+        public int Units = 0;
+
+        /// <summary>
+        /// The name of the player that owns this country.
+        /// </summary>
+        public string OwnedByName = "";
+
+        /// <summary>
+        /// The graphical X coordinate of the center of this country.
+        /// </summary>
+        public int X = 0;
```

Comment:
Could consolidate X and Y coordinates using Tuple class. Basically does the same thing but looks fancier. (+5 Style Points)